docs/src/man/split_apply_combine.md

```diff
@@ -138,3 +138,15 @@ julia> for subdf in groupby(iris, :Species)
 50
 50
 ```
+
+To also get the values of the grouped columns along with each group, use the
```

Comment:
@nalimilan - you are the master of docstring reviews :), but should we say here "grouped columns" or "grouping columns"? Also maybe we can mention that the values are stored in a `NamedTuple`?

Reply:
Yes, "grouping columns" sounds better. Mentioning `NamedTuple` also sounds like a good idea.